resources/views/Widgets/OrderConfirmation/OrderDocumentsWidget.twig

```diff
@@ -0,0 +1,80 @@
+{% import "Ceres::Widgets.Helper.TwigBuilder" as Twig %}
+{% import "Ceres::Widgets.Helper.WidgetHelper" as WidgetHelper %}
+
+{% set customClass = widget.settings.customClass.mobile %}
+{% set spacingSettings = widget.settings.spacing %}
+{% set inlinePadding   = WidgetHelper.getInlineSpacings(spacingSettings, spacingSettings.customPadding.mobile, "") %}
+{% set inlineMargin    = WidgetHelper.getInlineSpacings(spacingSettings, "", spacingSettings.customMargin.mobile) %}
+{% set spacingPadding  = WidgetHelper.getSpacingClasses(spacingSettings, spacingSettings.customPadding.mobile, "") %}
+{% set spacingMargin   = WidgetHelper.getSpacingClasses(spacingSettings, "",spacingSettings.customMargin.mobile) %}
+
+{% set appearance = widget.settings.appearance.mobile | default("primary") %}
+{% set buttonSize = widget.settings.buttonSize.mobile | default("") %}
+
+{{ Twig.set("enabledRoutes", Twig.call("config", ["IO.routing.enabled_routes"]) ~ " | split(\", \")") }}
+{{ Twig.set("translationMap",
+    {
+        invoice:            "Ceres::Template.orderConfirmationOrderDocumentsInvoice",
+        invoice_external:   "Ceres::Template.orderConfirmationOrderDocumentsInvoiceExternal",
+        delivery_note:      "Ceres::Template.orderConfirmationOrderDocumentsDeliveryNote",
+        order_confirmation: "Ceres::Template.orderConfirmationOrderDocumentsOrderConfirmation",
+        pickup_delivery:    "Ceres::Template.orderConfirmationOrderDocumentsPickupDelivery",
+        reversal_document:  "Ceres::Template.orderConfirmationOrderDocumentsReversalDocument"
+    } | json_encode)
+}}
+
+{{ Twig.set("orderDocuments", "[]") }}
+{{ Twig.for("document", "data.order.documents | default([])", "translationMap[document.type] is defined") }}
+    {{ Twig.set("orderDocuments", "orderDocuments | merge([document])") }}
+{{ Twig.endfor() }}
+
+{{ Twig.if("('order-document' in enabledRoutes or 'all' in enabledRoutes) and (orderDocuments | length > 0 or #{ isPreview | json_encode })") }}
```

Comment:
Warum wird hier `isPreview` abgefragt, wenn im elseif-Zweig für die Vorschau ein Platzhalter gerendert wird?

Reply:
Der else-Zweig ist nur für den Fall, dass die Route nicht aktiv ist und man sich in der Vorschau befindet.